packages/dso-toolkit/src/components/modal/modal.scss

```diff
@@ -54,7 +55,8 @@ dso-modal {
     .dso-tertiary {
       float: inline-start;
       line-height: calc(1.5em - 1px);
-      padding: button.$vertical-padding 0;
+      padding-block: button.$block-padding;
```

Comment:
Ik zie dat je dit op meer plekken hebt gedaan: -block en -inline separaat. Waarom? Kan het niet anders? Of voor de duidelijkheid?

Is dit niet meer mogelijk: `padding: button.$block-padding 0`?

Reply:
omdat de browsers dit verschillend oppakken, is het beter om ze te scheiden. de echte shorthand gebruiken met 3 of 4 values is echt niet meer mogelijk, omdat die dan 'physical' worden gerenderd.
Bijvoorbeeld: `padding: 1rem 2rem 3rem 4rem' wordt top-right-bottom-left gerenderd. logical moet het dus zo:
`padding-block: 1rem 3rem;
padding-inline: 4rem 2rem;`
^^ let op de omkering in volgorde bij `inline`; dat moet ivm start/end.
Gelukkig hoeven we dit maar 1 keer om te zetten, daarna doen we het gewoon goed ;)